config/routes.rb

```diff
@@ -10,6 +10,7 @@
           post 'datatosign'
           post 'sign'
           get 'parameters'
+          get 'validation'
```

Comment:
nie je lepsie skor sloveso `validate` ?

Reply:
@jsuchal neviem, bojoval som s tým. Je to GET metóda. Ale asi dáva zmysle spraviť POST /validate

Reply:
@celuchmarek v rails je bezne aj sloveso na GET. napr `show`, `index`, `new` pokial to nerobi nejake zapisy do db/upravu tak GET dava vacsi zmysel.